### PR TITLE
docs: Add docs about Cube Cloud IDE

### PR DESCRIPTION
**Pull request by keydunov**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: docs/content/Cube-Cloud/Cube-IDE.md [hunk 1]
```diff
@@ -10,3 +10,23 @@ browser. Cube Cloud can create branch-based develolment API instances to test ch
 in your frontend applications before pushing them into production.
 
 ![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Cube-Cloud/cube-ide.png)
+
+## Development Mode
+
+In development mode you can safely make changes to your project without affecting production deployment. Developer mode uses a separate Git branch and allows to test your changes in Playground or via Dev API endpoint specific to this branch. The Dev API hot-reloads your schema changes, allowing you to quickly test API changes from your applications.
```
Comment (rpaik) [inline docs/content/Cube-Cloud/Cube-IDE.md:16]:
```suggestion
In development mode, you can safely make changes to your project without affecting production deployment. Developer mode uses a separate Git branch and allows testing your changes in Playground or via Dev API endpoint specific to this branch. The Dev API hot-reloads your schema changes, allowing you to quickly test API changes from your applications.
```

**Review by rpaik — APPROVED** [review]

File: docs/content/Cube-Cloud/Cube-IDE.md [hunk 2]
```diff
@@ -10,3 +10,23 @@ browser. Cube Cloud can create branch-based develolment API instances to test ch
 in your frontend applications before pushing them into production.
 
 ![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Cube-Cloud/cube-ide.png)
+
+## Development Mode
+
+In development mode, you can safely make changes to your project without affecting production deployment. Developer mode uses a separate Git branch and allows testing your changes in Playground or via Dev API endpoint specific to this branch. The Dev API hot-reloads your schema changes, allowing you to quickly test API changes from your applications.
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Cube-IDE.md:16]:
```suggestion
In development mode, you can safely make changes to your project without affecting production deployment. Development mode uses a separate Git branch and allows testing your changes in Playground or via a separate API endpoint specific to this branch. This development API hot-reloads your schema changes, allowing you to quickly test API changes from your applications.
```

File: docs/content/Cube-Cloud/Cube-IDE.md [hunk 3]
```diff
@@ -10,3 +10,23 @@ browser. Cube Cloud can create branch-based develolment API instances to test ch
...
+
+To enter into the development mode navigate to the **Schema** page and click **Enter
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Cube-IDE.md:18]:
```suggestion
To enter development mode, navigate to the **Schema** page and click **Enter
```

File: docs/content/Cube-Cloud/Cube-IDE.md [hunk 4]
```diff
@@ -10,3 +10,23 @@ browser. Cube Cloud can create branch-based develolment API instances to test ch
...
+Development Mode**.
+
+![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Cube-Cloud/enter-dev-mode.png)
+
+When you enter the development mode you can see the grey bar on top. It provides
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Cube-IDE.md:23]:
```suggestion
When development mode is active, a grey bar will be visible on the top of the screen. It provides
```

File: docs/content/Cube-Cloud/Cube-IDE.md [hunk 5]
```diff
@@ -10,3 +10,23 @@ browser. Cube Cloud can create branch-based develolment API instances to test ch
...
+several useful controls and indicators:
+* It displays the name of the current development Git branch.
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Cube-IDE.md:25]:
```suggestion
* The name of the current development Git branch
```

File: docs/content/Cube-Cloud/Cube-IDE.md [hunk 6]
```diff
@@ -10,3 +10,23 @@ browser. Cube Cloud can create branch-based develolment API instances to test ch
...
+* It shows the status of the dev API. When you make changes to the project files the
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Cube-IDE.md:26]:
```suggestion
* The status of the development API. After any changes to the project, the
```

**Review by hassankhan — COMMENTED** [review]
Looks great :raised_hands: Just some minor edits 😄

**Comment by keydunov** [comment]
Thanks, @hassankhan @rpaik !

**Pull request merged**